elements/lisk-p2p/src/p2p.ts

```diff
@@ -779,18 +788,17 @@ export class P2P extends EventEmitter {
 			.minimumPeerDiscoveryThreshold
 			? this._config.minimumPeerDiscoveryThreshold
 			: DEFAULT_MIN_PEER_DISCOVERY_THRESHOLD;
-		const maximumPeerDiscoveryResponseSize = this._config
-			.maximumPeerDiscoveryResponseSize
-			? this._config.maximumPeerDiscoveryResponseSize
-			: DEFAULT_MAX_PEER_DISCOVERY_RESPONSE_SIZE;
+		const peerDiscoveryResponseLength = this._config.peerDiscoveryResponseLength
```

Comment:
I think we can leave `maximum` in the name here, as it describes the maximum and not the actual response length.

Reply:
@mitsujutsu I separated those two. I renamed the variables. The max length of the peer list that we accept from peers can be different from the length that we provide to peers. I'm worried that if we don't separate those it could make updates more difficult in the future.